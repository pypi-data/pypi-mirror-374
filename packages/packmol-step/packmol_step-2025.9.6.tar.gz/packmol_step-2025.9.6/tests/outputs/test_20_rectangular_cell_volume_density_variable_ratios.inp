tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside box 1.0000 1.0000 1.0000 8.0000 18.0000 28.0000
   number 25
end structure
structure input_2.pdb
   inside box 1.0000 1.0000 1.0000 8.0000 18.0000 28.0000
   number 74
end structure
