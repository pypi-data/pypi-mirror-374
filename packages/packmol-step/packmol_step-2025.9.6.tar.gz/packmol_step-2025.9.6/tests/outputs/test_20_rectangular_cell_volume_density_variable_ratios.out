    Will create a rectangular periodic cell containing the following molecules:

        +-------------+-------------+---------+
        |  Component  | Structure   |   Ratio |
        |-------------+-------------+---------|
        |    fluid    | c1ccccc1    |       1 |
        |    fluid    | O           |       3 |
        +-------------+-------------+---------+

    The dimensions of the region will be calculated from the volume 6.0 nm³. The
    number of molecules of the fluid will be obtained  by using the density 0.9
    g/ml.

Created a periodic 10.00 x 20.00 x 30.00 Å cell with 99 fluid molecules

    +-------------+-------------+---------------+----------+------------+
    |  Component  | Structure   |   Requested % |   Number |   Actual % |
    |-------------+-------------+---------------+----------+------------|
    |    fluid    | c1ccccc1    |            25 |       25 |     25.253 |
    |    fluid    | O           |            75 |       74 |     74.747 |
    +-------------+-------------+---------------+----------+------------+

There are a total of 522 atoms in the cell giving a density of 0.9094 g/ml.